src/preferences/dialog/dlgprefinterfacedlg.ui

```diff
@@ -287,16 +287,26 @@
       <item row="5" column="1" colspan="2">
        <widget class="QComboBox" name="comboBoxScreensaver"/>
       </item>
-      <item row="5" column="0">
+      <item row="6" column="0">
        <widget class="QLabel" name="multiSamplingLabel">
         <property name="text">
          <string>Multi-Sampling</string>
         </property>
        </widget>
       </item>
-      <item row="5" column="1" colspan="2">
+      <item row="6" column="1" colspan="2">
        <widget class="QComboBox" name="multiSamplingComboBox"/>
       </item>
+      <item row="7" column="0" colspan="2">
+       <widget class="QCheckBox" name="checkBoxForceHardwareAcceleration">
+        <property name="text">
+         <string>Force 3D acceleration</string>
+        </property>
+        <property name="toolTip">
+         <string>If enabled, Mixxx will try to detect whether 3D acceleration is available instead.</string>
+        </property>
```

Comment:
That sounds more like a tooltip for "enable 3D acceleration", not "force 3D acceleration"?